app/queries/my_facilities/blood_pressure_control_query.rb

```diff
@@ -100,7 +122,7 @@ def quarterly_uncontrolled_bps
   def monthly_registrations
```

Comment:
Same here, this is not counting registrations anymore

Reply:
This is actually the total registrations in the month at the facilities, so `monthly_registrations` makes sense.

Reply:
We're using `assigned_facility` to group this though. Is that the correct field to use for calculating registrations at a facility?

Reply:
Right, the `.where(:assigned_facility: facilities)` is incorrect indeed. Addressed in 23d8339.